common/djangoapps/student/models.py

```diff
@@ -1002,7 +1005,6 @@ def __unicode__(self):
         ).format(self.user, self.course_id, self.created, self.is_active)
 
     @classmethod
-    @transaction.atomic
```

Comment:
I have removed `@transaction.atomic`, and by default `ATOMIC_REQUESTS = False`, do I need to add `transaction.non_atomic_requests`